packages/web/src/migrations/update-13-0-0/remove-node-sass-13-0-0.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  formatFiles,
+  GeneratorCallback,
+  readJson,
+  removeDependenciesFromPackageJson,
+  Tree,
+} from '@nrwl/devkit';
+
+/**
+ * For gatsby app with style scss option, replace node-sass with sass in package.json
```

Comment:
for this comment, i think you meant to all web app?

Reply:
ah yes